ruby-webapp-onbuild/Dockerfile

```diff
@@ -24,6 +25,5 @@ ONBUILD COPY Gemfile.lock /usr/src/app/
 ONBUILD RUN socat UNIX-LISTEN:$SSH_AUTH_SOCK,fork TCP4:$(ip route|awk '/default/ {print $3}'):$SSH_AUTH_PROXY_PORT & bundle install
 
 ONBUILD COPY . /usr/src/app
-ONBUILD RUN mkdir /usr/src/app/public/assets
```

Comment:
To make other containers and processes happier can we change this to `mkdir -p …`